subt_ros/package.xml

```diff
@@ -24,6 +24,7 @@
   <exec_depend>compressed_image_transport</exec_depend>
   <exec_depend>message_runtime</exec_depend>
   <exec_depend>topic_tools</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
```

Comment:
This should probably go in `<depend>`.

Reply:
Without this, catkin gives the following error:

```
CMake Error at /opt/ros/melodic/share/catkin/cmake/catkin_package.cmake:224 (message):
  catkin_package() DEPENDS on the catkin package 'std_msgs' which must
  therefore be listed as a run dependency in the package.xml
Call Stack (most recent call first):
  /opt/ros/melodic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  subt/subt_ros/CMakeLists.txt:33 (catkin_package)
```